src/DataAbstractionLayer/Entity/WebhookLog/PayonePaymentWebhookLogEntity.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PayonePayment\DataAbstractionLayer\Entity\WebhookLog;
+
+use Shopware\Core\Checkout\Order\OrderEntity;
+use Shopware\Core\Framework\DataAbstractionLayer\Entity;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityIdTrait;
+
+class PayonePaymentWebhookLogEntity extends Entity
```

Comment:
i don't think setters are required, because you are not using these methods.  so please remove them.

Reply:
Done